[BOUNTY] Cassette Crates + More!

Noot-Toot opened the pull request:
## About The Pull Request
[completes this bounty](https://discord.com/channels/748354466335686736/1197311496015847485/1197311496015847485)
adds 4 crates + 3 goodies:
Crates:

1. Random Cassette Crate: contains 10 random, approved cassettes (presumably)
2. Blank Cassette Crate: contains 10 blank cassettes
3. Walkman Crate: contains 5 walkmans.
4. Cassette Deck Crate: Contains a cassette deck and a wrench.

Goodies:

1. Random Cassettes: 3 random cassettes
2. Blank Cassettes: 3 blank cassettes
3. Walkman: 1 walkman.

really they just wanted the first one but i wanted the other 3.
should probably be test-merged first as i couldnt really properly test if the random cassettes would work locally.
## Why It's Good For The Game
its good for my wallet, that is, if it wasnt 1 rounds worth of monkecoin.
## Changelog
:cl: Tractor Mann
add: Added 4 new crates concerning cassettes at cargo! (Most importantly the 10 Random Cassettes one, go wild with buying the cassettes, curators!)
add: 3 new cargo goodies! (if you need 3 cassettes instead of 10, or just 1 walkman.)
/:cl:

Camriod commented:
Have you considered making a personal order variant for crew requests? Forcing bulk orders from cargo for music doesn't sound like it'd be the easiest thing to get from the average tech.

Our recent flood of people has drastically changed out cargo culture, often being closed off and hostile. (Really not as much of a problem now but every now and then...)

Would be nice is all.

Noot-Toot commented:
> Have you considered making a personal order variant for crew requests? Forcing bulk orders from cargo for music doesn't sound like it'd be the easiest thing to get from the average tech.
> 
> The recent flood of people has drastically changed our cargo culture, often being closed off and hostile. (Really not as much of a problem now but every now and then...)
> 
> Would be nice is all.

added and pushed commit, should be working and ive hopefully not destroyed everything. also added a goodie.dm file for goodies since the old goodies were just in random files
